src/components/BridgeForm.js

```diff
@@ -49,7 +49,7 @@ const BridgeForm = ({ isOpen, onClose, bridge }) => {
         } else {
             setCandidatePorts(Object.values(interfaces));
         }
-    }, [interfaces]);
+    }, [bridge, isEditing, interfaces]);
```

Comment:
I'm not sure about this, but let's check it later.

Reply:
eslint was complying so, the same with bonding, I'm fine discarding the change

Reply:
I know. It was just a sort of self-note to check it on a near future. Keep the change, no problem.